Sources/DeepLearning/Operators.swift

```diff
@@ -133,6 +133,35 @@ public extension Padding {
     }
 }
 
+/// An older padding scheme. Used by padding, convolution, and pooling ops.
+// @_frozen // SR-9739
+public enum PaddingV1 {
```

Comment:
We should not introduce versioned types in the deep learning library. Why is this necessary? All ops should use the general `Padding` type.

Also, I don't think the extension below compiles. Could you verify things compile and work locally?

Reply:
The pooling ops do not support explicit paddings. We could either not support explicit paddings at all for now (i.e., not even for convolution ops), or have two different padding modes.

I checked and it compiled locally but I may have mixed up my branches. I'll check again.

Reply:
In that case, let’s fall back to not supporting explicit paddings for now.

Reply:
I just pushed changes removing the explicit padding mode.